readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -93,6 +93,13 @@ def test_import_repo_url(self):
                 form = ProjectBasicsForm(initial)
                 self.assertEqual(form.is_valid(), valid, msg=url)
 
+    def test_empty_slug(self):
+        initial = {
+            'name': "''",
+        }
+        form = ProjectBasicsForm(initial)
+        self.assertFalse(form.is_valid())
+
```

Comment:
We should check that the error really comes from `name`

Reply:
yeah, you are right.

In fact, I forget to add the rest of the fields in the initial data.